core_info.c

```diff
@@ -302,6 +303,14 @@ static core_info_list_t *core_info_list_new(const char *path,
             tmp = NULL;
          }
 
+         if (config_get_string(conf, "rpcname", &tmp)
```

Comment:
Seems like we could use `rpcname` elsewhere... Seems similar to the `systemname`, so possibly rename it to `systemid`?